.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,36 +1,37 @@
-### Related issue(s) and PR(s)
-This PR closes [issue number] and relates to [issue or PR number]
+**Related issue(s) and PR(s)**
 
```

Comment:
Maybe remove this line since (what I interpret) there is no gap between headings and text for the other sections?
```suggestion
```

Reply:
Good that you spotted that! It wasn't enough to remove that line; I also had to add two spaces at the end of the previous one to avoid GitHub merging the lines.